packages/console/src/components/AppContent/components/Sidebar/components/Item/index.tsx

```diff
@@ -16,12 +17,37 @@ const Item = ({ icon, titleKey, isActive = false }: Props) => {
   const { t } = useTranslation(undefined, {
     keyPrefix: 'admin_console.tabs',
   });
+  const [isContactOpen, setIsContactOpen] = useState(false);
+  const isContactUs = titleKey === 'community_support';
```

Comment:
`community_support` should be `contact_us`?

Reply:
yep, will change it after rebasing #505